exporter/googlemanagedprometheusexporter/config.go

```diff
@@ -0,0 +1,126 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlemanagedprometheusexporter
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	gmp "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlemanagedprometheusexporter"
+	"go.opentelemetry.io/collector/component"
+	"go.uber.org/multierr"
+	"google.golang.org/api/option"
+)
+
+const (
+	defaultUserAgent = "observIQ-otel-collector"
+)
+
+// Config is the config the google managed prometheus exporter
+type Config struct {
+	Credentials     string      `mapstructure:"credentials"`
+	CredentialsFile string      `mapstructure:"credentials_file"`
+	GMPConfig       *gmp.Config `mapstructure:",squash"`
+}
+
+// Validate validates the config
+func (c *Config) Validate() error {
+	var err error
+	err = multierr.Append(err, c.GMPConfig.Validate())
```

Comment:
nit: Do we need to use multi err here as there's only one error?

Reply:
Nope, totally missed that as I was cutting out some of the logic. Good catch!